app/templates/partials/flash_messages/archive_service_confirmation_message.html

```diff
@@ -0,0 +1,18 @@
+{% if platform_admin %}
+      <h2 class="banner-title">Are you sure you want to delete ‘{{service_name}}’?</h2>
+      <br>
+      <h3 class="govuk-body">Before you delete ‘{{service_name}}’: </h3>
```

Comment:
I think this can just be a paragraph as there is already a convention with the leading paragraph before a list being a kind of label for the points.

```suggestion
      <p class="govuk-body">Before you delete ‘{{service_name}}’: </p>
```